_docs/concepts/security/authn-policy.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Istio authentication policy
+overview: Describes Istio Authentication policy
+
+order: 10
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+## Overview
+Istio authentication policy enables admin to specify authentication requirements for a service (or services). Istio authentication policy is composed of two-part authentication:
+
+* Peer: verifies the party, the direct client, that makes the connection. The common authentication mechanism for this is [mutual TLS]({{home}}/docs/concepts/security/mutual-tls.html). Istio will be responsible to manage both client and server sides to enforce the policy.
```

Comment:
responsible to manage -> responsible for managing

Reply:
Done